lib/mapper.js

```diff
@@ -144,9 +163,9 @@ exports.group = function(msg) {
     remote_created_at: time(msg.created()),
     company_id: msg.groupId(),
     name: msg.name(),
-    monthly_spend: msg.proxy('traits.monthlySpend'),
+    monthly_spend: msg.proxy('traits.monthlySpend') || msg.proxy('traits.monthly_spend'),
```

Comment:
You shouldn't need to specify both of these casings, since `proxy` handles all the different casings of the input.